fixtures/elation/fuze-par-z60ip.json

```diff
@@ -0,0 +1,771 @@
+          "type": "Generic",
+          "comment": "Color Preset 43"
+        },
+        {
+          "dmxRange": [173, 176],
+          "type": "Generic",
+          "comment": "Color Preset 44"
+        },
+        {
+          "dmxRange": [177, 180],
+          "type": "Generic",
+          "comment": "Color Preset 45"
+        },
+        {
+          "dmxRange": [181, 184],
+          "type": "Generic",
+          "comment": "Color Preset 46"
+        },
+        {
+          "dmxRange": [185, 188],
+          "type": "Generic",
+          "comment": "Color Preset 47"
+        },
+        {
+          "dmxRange": [189, 192],
+          "type": "Generic",
+          "comment": "Color Preset 48"
+        },
+        {
+          "dmxRange": [193, 196],
+          "type": "Generic",
+          "comment": "Color Preset 49"
+        },
+        {
+          "dmxRange": [197, 200],
+          "type": "Generic",
+          "comment": "Color Preset 50"
+        },
+        {
+          "dmxRange": [201, 204],
+          "type": "Generic",
+          "comment": "Color Preset 51"
+        },
+        {
+          "dmxRange": [205, 208],
+          "type": "Generic",
+          "comment": "Color Preset 52"
+        },
+        {
+          "dmxRange": [209, 212],
+          "type": "Generic",
+          "comment": "Color Preset 53"
+        },
+        {
+          "dmxRange": [213, 216],
+          "type": "Generic",
+          "comment": "Color Preset 54"
+        },
+        {
+          "dmxRange": [217, 220],
+          "type": "Generic",
+          "comment": "Color Preset 55"
+        },
+        {
+          "dmxRange": [221, 224],
+          "type": "Generic",
+          "comment": "Color Preset 56"
+        },
+        {
+          "dmxRange": [225, 228],
+          "type": "Generic",
+          "comment": "Color Preset 57"
+        },
+        {
+          "dmxRange": [229, 232],
+          "type": "Generic",
+          "comment": "Color Preset 58"
+        },
+        {
+          "dmxRange": [233, 236],
+          "type": "Generic",
+          "comment": "Color Preset 59"
+        },
+        {
+          "dmxRange": [237, 240],
+          "type": "Generic",
+          "comment": "Color Preset 60"
+        },
+        {
+          "dmxRange": [241, 244],
+          "type": "Generic",
+          "comment": "Color Preset 61"
+        },
+        {
+          "dmxRange": [245, 248],
+          "type": "Generic",
+          "comment": "Color Preset 62"
+        },
+        {
+          "dmxRange": [249, 252],
+          "type": "Generic",
+          "comment": "Color Preset 63"
+        },
+        {
+          "dmxRange": [253, 255],
+          "type": "Generic",
+          "comment": "Color Preset 64"
+        }
+      ]
+    },
+    "Program Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [1, 20],
+          "type": "Generic",
```

Comment:
Same with the macros, no information on the programs.

Reply:
Please set the capability type to `Effect` and add a help wanted message here, too.